src/ucp/api/ucp.h

```diff
@@ -3259,8 +3259,7 @@ ucs_status_ptr_t ucp_put_nb(ucp_ep_h ep, const void *buffer, size_t length,
  *
  * @param [in]  ep           Remote endpoint handle.
  * @param [in]  buffer       Pointer to the local source address.
- * @param [in]  length       Length of the data (in bytes) stored under the
- *                           source address.
+ * @param [in]  count        Number of elements to put.
```

Comment:
Majority of our user probably use it to send bytes. So probably you want to put that for default datatype (contig?) it corresponds to bytes.

Reply:
agree, this is the case today: https://github.com/openucx/ucx/blob/master/src/ucp/api/ucp.h#L1270

Reply:
@shamisp is this good to go? it's needed for v1.9.x branch

Reply:
@yosefe you agreed but the text was not updated ?

Reply:
@shamisp there is no need to update the text because it's already documented: https://github.com/openucx/ucx/blob/master/src/ucp/api/ucp.h#L1270

Reply:
@yosefe Maybe I'm missing something.  it is bad practice to send developer to another structure (ucp_request_param_t) to understand meaning for the function arguments (count), when it developer actually does need to set ucp_request_param_t & datatype at all. Moreover it is nowhere spell out that count == bytes for ucp_dt_make_contig.

My suggestion is the following, change:  Number of elements to put
to: Number of elements to put. For default datatype @ref ucp_dt_make_contig  count corresponds to bytes. For any other  datatypes specified in  @ref ucp_request_param_t  the argument correspond for number of element for this datatype.

Reply:
@shamisp count is the number of elements. ucp_request_param_t.datatype is type of each element, type which defaults to "byte" if not specified. 
we can write `@param [in]  count        Number of elements of type @ref ucp_request_param_t.datatype to put.`

Reply:
@yosefe This is an argument the developer even does not have to set. It is bad idea to send developer to read a documentation of the argument that by default is not going to be used. Since default is contiguous which just have to spell this out.

Reply:
ok, i see your point. how about:
`@param [in] count Number of elements of type @ref ucp_request_param_t.datatype to put. If @ref ucp_request_param_t.datatype is not specified, the type defaults to ucp_dt_make_contig(1).`

Reply:
```@param [in] count Number of elements of type @ref ucp_request_param_t.datatype to put. If @ref ucp_request_param_t.datatype is not specified, the type defaults to ucp_dt_make_contig(1), which corespondent to byte elements.``` ?

Reply:
LGTM
@hoopoepg can you pls change to the above (for both put and get)

Reply:
@yosefe done
squash after tests are passed?

Reply:
yes
@shamisp is it ok now?

Reply:
Yep, looks good.